app-center-webapps/src/main/webapp/vue-apps/adminSetup/components/AdminSetupList.vue

```diff
@@ -64,7 +64,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
               v-exo-tooltip.bottom.body="props.item.title.length > 22 ? props.item.title : ''"
               class="text-md-center tableAppTitle"
             >
-              {{ props.item.displayName ? props.item.displayName : props.item.title}}
+              {{ props.item.displayName ? props.item.displayName : props.item.title }}
```

Comment:
Is this change needed ?

Reply:
It is a warning on npm watch so i fixed it.